.travis.yml

```diff
@@ -1,12 +1,6 @@
-language: crystal
+services:
+  - docker
 
 script:
-  - crystal tool format --check
-  - crystal spec ./spec/core
-  - rm shard.yml && rm shard.lock && cp shard.amber.yml ./shard.yml && shards install
-  - crystal spec ./spec/helpers/amber_spec.cr
-  - crystal spec ./spec/providers/amber_spec.cr
-  - rm shard.yml && rm shard.lock && cp shard.lucky.yml ./shard.yml && shards install
-  - crystal spec ./spec/helpers/lucky_spec.cr
-  - crystal spec ./spec/providers/lucky_spec.cr
+  - docker build ./ -t openapi_gen && docker run openapi_gen
```

Comment:
I'm wondering, is `docker run openapi_gen` needed since the specs are run during the build? It seems like `docker build` returns with an exit code of 0 when the specs pass.

Reply:
I'll merge anyway, it is not very important.